docs/development/architecture.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Architecture
+sidebar_position: 0
+---
+
+
+## Introduction
+
+Headlamp is a user-friendly and extensible Kubernetes UI designed to provide a seamless experience for managing Kubernetes clusters. It can connect to one or multiple Kubernetes clusters and can be deployed as a web app in-cluster or as a desktop app. Headlamp can be customized and 
+
+## High-Level Architecture
+
+### Headlamp running as a Desktop App (on MacOS, Windows, Linux etc)
+```mermaid
```

Comment:
This is not displaying in the website.

Reply:
docusaurus can support it with a plugin https://docusaurus.io/docs/markdown-features/diagrams